resultsdbupdater/consumer.py

```diff
@@ -1,9 +1,8 @@
-import fedmsg.consumers
-import fedmsg.config
+import fedmsg
```

Comment:
This change in the import causes the the class definition on line 12 to fail (you can see this in the CI run). Please revert the imports back.

Reply:
Ack